webapp/channels/src/components/user_group_popover/group_member_list/group_member_list.tsx

```diff
@@ -149,6 +155,8 @@ const GroupMemberList = (props: Props) => {
     };
 
     const Item = ({index, style}: ListChildComponentProps) => {
+        const status = useSelector((state: GlobalState) => getStatusForUserId(state, members[index].user.id) || UserStatuses.OFFLINE);
```

Comment:
```suggestion
        const status = useSelector((state: GlobalState) => getStatusForUserId(state, members[index]?.user?.id) || UserStatuses.OFFLINE);
```

I pull your changes and it was breaking when clicking to see the group information. I identified this line as the problem, with this optional values the issue is solved 
<img width="639" alt="Screenshot 2024-08-07 at 23 58 51" src="https://github.com/user-attachments/assets/26f18b0e-e5ed-4041-8900-409780967396">